languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -728,7 +728,19 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_HOMONYMES_PARONYMES" name="Confusion d’homonymes et paronymes">
-    <rule id="GENTE_GENT" name="gente → gent">
+      <rule id="LE_COVID" name="la Covid et le Covid" type="style" tags="picky" default="temp_off">
+          <pattern>
+              <marker>
+                  <token>le</token>
+              </marker>
+              <token regexp="yes">Covid-19|Covid|covid|covid-19</token>
```

Comment:
Unless you tell it to be case-sensitive, the string will be case-insensitive. This means you can simply write
`<token regexp="yes">covid(-19)?</token>`, thus covering all of the matches you wrote above. Plus the all-caps version COVID.